package.json

```diff
@@ -44,6 +44,16 @@
     ]
   },
   "devDependencies": {
-    "gh-pages": "^3.2.3"
+    "babel-eslint": "^10.1.0",
```

Comment:
npm **WARN** deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.

https://www.npmjs.com/package/@babel/eslint-parser